yaml/pois.yaml

```diff
@@ -548,7 +548,7 @@ filters:
       tier: 3
   # university
   - filter: {amenity: university}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.55 ] }, *tier3_min_zoom ] }, 15 ] }
+    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.55 ] }, *tier3_min_zoom ] }, 16 ] }
```

Comment:
Just noting for review that **college** was already set to default `min_zoom: 16` so no action was needed there.